sechub-pds-solutions/gosec/tests/clone_repo.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+vulnerable_apps=(
+    "https://github.com/globocom/secDevLabs.git",
+    "https://github.com/OWASP/crAPI"
+)
+
+current_test_folder="$1"
+vulnerable_repo="$2"
+	
+if [[ -z "$current_test_folder" ]]
```

Comment:
-z tests if the string has zero length. We should test if it exists and is a directory:
```suggestion
if [[ ! -d "$current_test_folder" ]]
```